04-simulate-an-election/election.py

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2016 Ami . All rights reserved
+
+from random import random
+
+def elect_in_region(election_result, candidate_chances):
+    if election_result <= candidate_chances:
+        return 'a'
+    else:
+        return 'b'
+
+def simulate_a_single_election():
+    candidate_a_chances = {
+        "region1": 87,
+        "region2": 65,
+        "region3": 17
+    }
+
+    candidate_a_elected_count = 0
+    candidate_b_elected_count = 0
+
+    for region in candidate_a_chances:
+        elected = elect_in_region(election_result=int(random() * 100), candidate_chances=candidate_a_chances[region])
```

Comment:
Also, please keep the 100 chars limit per line, and break longer lines according to the Python style guide.